### DISCUSSION TITLE
Прошивка из pvvx zigbee на devbis zigbee

### DISCUSSION THREAD

**Discussion by vladelen1** [devbis/z03mmc#151] [General]
Подскажите, пожалуйста, где взять прошивку devbis и метаданные к ней для перепрошивки по OTA из прошивки pvvx zigbee сразу на devbis zigbee через Z2M.

**Comment by vladelen1** [comment]
Например вот данные по прошивке от vppx:
{"fileVersion": 18952193,
        "fileSize": 127298,
        "manufacturerCode": 4417,
        "imageType": 522,
        "sha512": "7dbb388c19b2d80d184069243141771f938d560b16ca41b1cfc6daf9151ddb2038becdec8010ee2b24763ebdb92f96efb54f51f0e81764ab3e399a52a43c6725",
        "url": "https://github.com/pvvx/ZigbeeTLc/raw/master/bin/1141-020a-01213001-Z03MMC.zigbee"}
Без этих данных Z2M не подхватывает OTA прошивку.

**Comment by devbis** [comment]
You can use a file from here https://github.com/devbis/z03mmc/issues/118#issuecomment-1942371299
It is version 1.1.0 with pvvx's metadata

**Comment by vladelen1** [comment]
Не хочет устанавливатся.
1. В /homeassistant/zigbee2mqtt/configuration.yaml
```
ota:
  zigbee_ota_override_index_location: otaatc.json
  ikea_ota_use_test_url: false
  update_check_interval: 1

```
2. В /homeassistant/zigbee2mqtt/otaatc.json
```
[
  {
        "fileVersion": 18427905,
        "fileSize": 128034,
        "manufacturerCode": 4417,
        "imageType": 522,
        "url": "1141-020a-11003001-z03mmc.zigbee",
        "force": true
  }

]
```

Файл 1141-020a-11003001-z03mmc.zigbee находится в папке /homeassistant/zigbee2mqtt.

Результат:
![Безымянный](https://github.com/devbis/z03mmc/assets/163397503/8a6a4f28-da8a-465a-a847-6b2c85933c71)


Что я делаю не так?

PS Уже и так изменил содержимое (проставил SHA512, рассчитав в онлайн-калькуляторе и размер файла). Не помогло.

![Безымянный](https://github.com/devbis/z03mmc/assets/163397503/9d6bd0b5-9053-440c-bb1a-b1cd5a25b7eb)

```
[
  {
        "fileVersion": 18427905,
        "fileSize": 131362,
        "manufacturerCode": 4417,
        "imageType": 522,
        "url": "1141-020a-11003001-z03mmc.zigbee",
        "sha512": "6e3fbceec6a572a9084ae114da5843b1ebc67447c0d3b7fea5a7acff984260f1a3f1e400fe1a68b15925f334a5b42686baa6d3c0d853d9e1fdb8d0b4e801afc0",
        "force": true
  }

]
```

**Comment by vladelen1** [comment]
Вопрос снят. Прошился через USB-UART переходник.
